doc/installation_windows.md

```diff
@@ -1,18 +1,13 @@
-# Windows 8.1 & Windows 10 Installation
+# Windows 10 & Windows 11 Installation
 
 **Note:** Due to the USB 3.0 translation layer between native hardware and virtual machine, the librealsense team does not recommend or support installation in a VM.
 
-librealsense shall be built on Windows using [CMake v3.8+](https://cmake.org/download/) and Visual Studio 2015 / 2017: 
+librealsense shall be built on Windows using [CMake v3.8+](https://cmake.org/download/) and Visual Studio 2015 / 2017 / 2019: 
```

Comment:
2022 as well

Reply:
> Please verify the minimum cmake version , I dont think it's 3.8 today

I scanned entire repo and found that a lowest CMake version is 2.8.3. (third party dependency)
So I need write 2.8.3 in the guide?

Reply:
Which 3-rd party? seems too low.

Reply:
C:\Users\tyurovs\repos\librealsense\third-party\realsense-file\rosbag\config.cmake (1 hit)
	Line  1: cmake_minimum_required(VERSION 2.8...3.20.5)
  C:\Users\tyurovs\repos\librealsense\third-party\realsense-file\rosbag\console_bridge\config.cmake (1 hit)
	Line  1: cmake_minimum_required(VERSION 2.8...3.20.5)
  C:\Users\tyurovs\repos\librealsense\third-party\realsense-file\rosbag\cpp_common\config.cmake (1 hit)
	Line  1: cmake_minimum_required(VERSION 2.8.3...3.20.5)
  C:\Users\tyurovs\repos\librealsense\third-party\realsense-file\rosbag\rosbag_storage\config.cmake (1 hit)
	Line  1: cmake_minimum_required(VERSION 2.8.3...3.20.5)
  C:\Users\tyurovs\repos\librealsense\third-party\realsense-file\rosbag\roscpp_serialization\config.cmake (1 hit)
	Line  1: cmake_minimum_required(VERSION 2.8.3...3.20.5)
  C:\Users\tyurovs\repos\librealsense\third-party\realsense-file\rosbag\roscpp_traits\config.cmake (1 hit)
	Line  1: cmake_minimum_required(VERSION 2.8.3...3.20.5)
  C:\Users\tyurovs\repos\librealsense\third-party\realsense-file\rosbag\roslz4\config.cmake (1 hit)
	Line  1: cmake_minimum_required(VERSION 2.8.3...3.20.5)
  C:\Users\tyurovs\repos\librealsense\third-party\realsense-file\rosbag\rostime\config.cmake (1 hit)
	Line  1: cmake_minimum_required(VERSION 2.8.3...3.20.5)

Reply:
Make sure someone use those cmake files, I bet not.